source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
+    * Reconstructed event X-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``DETY`` tform: ``1E``, unit: deg
+    * Reconstructed event Y-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``EVENT_TYPE`` tform: ``32X``
+    * Event quality partition.
+
+      .. note::
+         It needs to be discussed whether this should be a mandatory column
+         or an optional column. The partition of events into quality classes
+         seems sufficiently fundamental so that all IACT should in principle
+         support this. For example event multiplicity could be translated
+         into quality partitions, and the IRFs computed accordingly.
+
+
+Optional columns
+----------------
+
+.. note::
+   None of the following columns is required to be part of an ``EVENTS``
+   extension. Any software using these columns should first check whether the
+   columns exist, and warn in case of their absence.
+
+* ``ALT`` tform: ``1E``, unit: deg
+    * Reconstructed altitude coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``AZ`` tform: ``1E``, unit: deg
+    * Reconstructed azimuth coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``THETA`` tform: ``1E``, unit: deg
+    * Reconstructed offset from the observation pointing position
+* ``PHI`` tform: ``1E``, unit: deg
+    * Reconstructed position angle from the observation pointing position
+      (position angles are counted counterclockwise from celestial North)
+* ``MULTIP`` tform: ``1I``
+    * Telescope multiplicity. Number of telescopes that have seen the event
+* ``DIR_ERR`` tform: ``1E``, unit: deg
+    * Direction error of reconstruction 
+* ``ENERGY_ERR`` tform: ``1E``, unit: TeV
+    * Error on reconstructed event energy
+* ``COREX`` tform: ``1E``, unit: m
+    * Core position X of shower
+* ``COREY`` tform: ``1E``, unit: m
+    * Core position Y of shower
+* ``CORE_ERR`` tform: ``1E``, unit: m
+    * Error on core position of shower    
+* ``XMAX`` tform: ``1E``, unit: radiation lengths
+    * First interaction depth 
+* ``XMAX_ERR`` tform: ``1E``, unit: radiation lengths
+    * Error on first interaction depth 
+* ``HIL_MSW`` tform: ``1E``
+    * Hillas mean scaled width
+* ``HIL_MSW_ERR`` tform: ``1E``
+    * Hillas mean scaled width error
+* ``HIL_MSL`` tform: ``1E``
+    * Hillas mean scaled length
+* ``HIL_MSL_ERR`` tform: ``1E``
+    * Hillas mean scaled length error
+
+
+Mandatory header keywords
+-------------------------
```

Comment:
Hmm ... I see there's actually significant changes here that will break everything (all events lists produced so far are invalid and no science tools work with the new format specified here).

The most important change is that you've removed RA_PNT and DEC_PNT as event header keywords as key parameters used for analysis and moved them to the more flexible POINTING table.

I see this as the start of introducing required TECH3 that is separate from EVENT3.
I think this is a good move, although the question for me is whether we should do this now, quickly, or take our time and do it later.

Note that if you make this backward-incompatible change, it would probably good as well do also move other info out of the EVENTS FITS header, specifically I think it would be more flexible to have LIVETIME and DEADC info with the GTI info. So I see this as a half-done change to introduce TECH3, with some info left completely or duplicated in the EVENTS header (time info) and others removed (pointing info).

Probably we should either keep RA_PNT and DEC_PNT and backward compat here now, or go more all-in on big changes. @jknodlseder, before we ping others about this, what do you think?